src/model/embedding/noms2embeddings_model.py

```
@@ -26,37 +26,54 @@ def _create_and_save_embeddings(
+    print(f"Number of nomenclatures: {noms_count}")
 
-    job.meta['total_count'] = len(df_noms)
+    job.meta['total_count'] = noms_count
     job.meta['ready_count'] = 0
+
+    job.meta['general_status'] = "Extracting features"
     job.save_meta()
 
     # Извлечение характеристик и добавление их в метаданные
     df_noms_with_features = extract_features(df_noms)
     print(f"Nomenclatures with features:")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)